### ISSUE TITLE
Allow to specify initial working directory in container

### ISSUE THREAD

**Issue by olifre** [hpc/charliecloud#72]
Especially when `--no-home` is not passed, the user might expect to end up in his / her home directory. 
In general, it would be a nice shortcut to be able to specify the IWD in the container.

**Comment by reidpr** [comment]
I believe the current behavior is container `/`?

It's a reasonable feature request, though I believe we should consider it carefully because it goes (a little bit) beyond "just start a container".

Can you write a brief argument on why a command line option (e.g.) would be better than the current way to do this, which is:

```
$ ch-run /img/foo -- sh -c "cd bar && baz ..."
```

**Comment by olifre** [comment]
Yes, the current behaviour is `/` of the container. 

The use-case I had at hand was to allow HTCondor (when starting a container with a user-provided executable) to define an initial working directory without touching the actual user executable command, and without wrapping an additional `sh` around. 

So yes, it's mainly the "convenience" argument. If I have an executable which is living in and supposed to work in `/workdir`, reading input and writing output to that folder, it's significantly more straightforward to write:
```
$ ch-run -b /scratch/workdir:/workdir --iwd /workdir /img/foo -- ./myAnalysisJob --channel="H tt"
```
than
```
$ ch-run -b /scratch/workdir:/workdir /img/foo -- sh -c 'cd /workdir && ./myAnalysisJob -- channel="H tt"'
```
So in short, mostly convenience - and since I guess it's very common to choose a working directory other than `/ ` especially for computing tasks.

**Comment by trandles-lanl** [comment]
I'll second Oliver's convenience use case.  Coming from a slightly different angle, some applications expect to be able to write output to CWD.  If you're bind-mounting in something like a scratch filesystem, being able to land there without script-fu like "cd /my/scratch/filesystem && /path/to/executable" is easier to manage.  It should also make read-only container use simpler if you can specify a writable directory for --iwd.

**Comment by reidpr** [comment]
I'm convinced.

What if we call the argument `-c` or `--cd`? I think `--iwd` is a little opaque.

Oliver, any interest in a PR or should we find someone else?

**Comment by olifre** [comment]
`-c` and `--cd` sound nice to me, good idea!
I'll give a PR a go, likely tomorrow.
